climada/util/calibrate/base.py

```diff
@@ -0,0 +1,399 @@
+"""Calibration Base Classes and Interfaces"""
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass, field, InitVar
+from typing import Callable, Mapping, Optional, Tuple, Union, Any, Dict
+from numbers import Number
+
+import pandas as pd
+import numpy as np
+from scipy.optimize import Bounds, LinearConstraint, NonlinearConstraint
+import seaborn as sns
```

Comment:
TODO: Fix requirements

Reply:
@peanutfun which seaborn runs in your local environment? (addding seaborn from pypi just like that yields errors)

Reply:
@emanuel-schmid Currently `seaborn=0.13.0`

Reply:
@emanuel-schmid That was wrong, it only works with `seaborn>=0.13`, see the new Jenkins config